scripts/Create-StorageContainer.ps1

```diff
@@ -24,6 +24,9 @@ Name of the storage account where the container will be created
 .PARAMETER StorageContainerNames
 Name(s) of the storage container(s) to create. Use a comma-separated list to specify multiple containers
 
+.PARAMETER PublicStorageContainerNames
+Name(s) of the storage container(s) to create. Use a comma-separated list to specify multiple containers
```

Comment:
What's the difference between this parameter and StorageContainerNames? Reflect that in the comment as well